src/main/java/Controller.java

```diff
@@ -1658,6 +1696,44 @@ void handleNewOrder(ActionEvent event) {
         }
     }
 
+    @FXML
+    void assignDefaultFileLocation(ActionEvent event) {
+        String pathString = DefaultFileLocation.getText();
+        File temp = new File (pathString);
+        if (temp.exists() && temp != null) {
+            actiontarget.setText("The default file was set to: " + temp.getPath());
```

Comment:
Remove the default file was set to as this can be seen right after with current default file is. It seems repetitive to have both.

Reply:
Addressed, I set it to blank, but kept the message that is displayed when the input is an invalid path. So it resets and prints blank upon a valid path, but prints an error message upon an invalid path.